contracts/src/bridge/ISequencerInbox.sol

```diff
@@ -51,33 +52,104 @@ interface ISequencerInbox is IDelayedMessageProvider {
     /// @dev a keyset was invalidated
     event InvalidateKeyset(bytes32 indexed keysetHash);
 
+    function totalDelayedMessagesRead() external view returns (uint256);
+
+    function bridge() external view returns (IBridge);
+
+    /// @dev The size of the batch header
+    // solhint-disable-next-line func-name-mixedcase
+    function HEADER_LENGTH() external view returns (uint256);
+
+    /// @dev If the first batch data byte after the header has this bit set,
+    ///      the sequencer inbox has authenticated the data. Currently not used.
+    // solhint-disable-next-line func-name-mixedcase
+    function DATA_AUTHENTICATED_FLAG() external view returns (bytes1);
+
+    function rollup() external view returns (IOwnable);
+
+    function isBatchPoster(address) external view returns (bool);
+
+    // function maxTimeVariation() external view returns (MaxTimeVariation calldata);
```

Comment:
```suggestion
    function maxTimeVariation() external view returns (MaxTimeVariation calldata);
```

Reply:
seems like this isn't possible
https://github.com/ethereum/solidity/issues/11826

Reply:
added some comment